Blogplace.Web/Domain/Comments/Comment.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Blogplace.Web.Domain.Comments;
```

Comment:
bardziej Articles bo domenowo komentarz to część artykułu a nie coś osobnego i niezależnego